.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: required
 dist: trusty
 language: node_js
 node_js:
-- '6.9.1'
```

Comment:
It's just `>=` which `6.11.0` is. The newer `pr-bumper` scripts use `6.11.0` as the gate for publishing, so I figured we didn't need both `6.11.0` and `6.9.1` I really don't think we're using anything that's in `6.11.0` and not `6.9.1`, but if it makes you feel better, we can add that to the build matrix. I don't think we need to change the `engines` to move up to `6.11.0` though, since it should work just fine with `6.9.1` still.